### PR TITLE
Add ReleaseNotes for 2.9.6

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: ReleaseNotes-2.9.6 [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#
+# Open Build Service 2.9.6
+#
+
+Please read the README.SETUP file for initial installation
+instructions or use the OBS Appliance from
+
+  http://openbuildservice.org/download
+
+There is also an install medium which installs OBS on hard disc now.
```
Comment (dmarcoux) [inline ReleaseNotes-2.9.6]: hard dis**k**

File: ReleaseNotes-2.9.6 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+dist/README.UPDATERS file has informations for updaters.
```
Comment (dmarcoux) [inline ReleaseNotes-2.9.6]: information<strike>s</strike>

File: ReleaseNotes-2.9.6 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+OBS Appliance users who have setup their LVM can just replace
+their appliance image without data loss. The migration will
+happen automatically.
+
+Bugfixes
+========
+
+Frontend:
+  * Rails security update was patched
```
Comment (hennevogel) [inline ReleaseNotes-2.9.6]: Please mention the CVE.

File: ReleaseNotes-2.9.6 [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  * Added upper-limit to range to avoid long running queries in Webui::MonitorController
+  * Only admins are allowed to create DoD repositories
+  * Only admins are allowed to create sourceaccess/access repositories flags
```
Comment (hennevogel) [inline ReleaseNotes-2.9.6]: both we only changed in the webui. please mention this.

**Comment by DavidKang** [comment]
@hennevogel and @dmarcoux, could you review it again please? :smile_cat:

**Pull request merged**